src/environment-d-generator/meson.build

```diff
@@ -13,6 +13,8 @@ executables += [
 ]
 
 install_emptydir(environmentdir)
-meson.add_install_script(sh, '-c',
-                         ln_s.format(sysconfdir / 'environment',
-                                     environmentdir / '99-environment.conf'))
+if install_sysconfdir
+        meson.add_install_script(sh, '-c',
+                                 ln_s.format(sysconfdir / 'environment',
+                                             environmentdir / '99-environment.conf'))
```

Comment:
Hmm? We're not installing anything into /etc/? Here a symlink is created under /usr/lib/environment.d/ which points to /etc/environment. Why should this be affected by install_sysconfdir??

Reply:
Thanks, you're totally right! I was confused as well.

Reply:
Also mentioned in the environment.d(5) manpage about th backward compat with /etc/environment